### PR TITLE
[fix] Remove some defaults in Schema.statics.search

### PR DESCRIPTION
**Pull request by siennamw**

### CODE REVIEW

File: mongoose/index.js [hunk 1]
```diff
@@ -68,19 +68,24 @@ const mongooseExtendOptions = (Schema) => {
...
+
+		if (options.populate) {
+			query = query.populate(options.populate);
+		}
+
+		if (options.projection) {
+			query = query.select(options.projection);
+		}
+
+		if (options.sortProperty && options.sortOrder) {
```
Comment (svet93) [inline mongoose/index.js:86]: Did you double check that these 2 are always used together?

Comment (siennamw) [reply mongoose/index.js:86]: If you look at the syntax below you'll see they are both necessary.  Could default to 1 on sortOrder, though.  Anybody else have thoughts on this?

Comment (svet93) [reply mongoose/index.js:86]: Yes, but there could be existing code that doesn't provide both if they counted on the 1 being set by default for order.

Comment (bearfact) [reply mongoose/index.js:86]: I think you want to maintain a default setting for sortOrder, but only apply the sort function if a sortProperty is passed.

**Comment by bearfact** [comment]
Lets find a few places in code where sorting, pagination, and select/projection are used and do a manual regression just to be extra careful.

**Comment by tom-con** [comment]
> Lets find a few places in code where sorting, pagination, and select/projection are used and do a manual regression just to be extra careful.

One place in particular is yesterday's fix were I just added `sortProperty` -- @svet93 do you remember where that was?

File: mongoose/index.js (hunk 1 above)
Comment (siennamw) [reply mongoose/index.js]: Okay, added in the default sort order

**Comment by bearfact** [comment]
:+1:

**Review by bearfact — APPROVED** [review]

**Comment by svet93** [comment]
> > Lets find a few places in code where sorting, pagination, and select/projection are used and do a manual regression just to be extra careful.
> 
> One place in particular is yesterday's fix were I just added `sortProperty` -- @svet93 do you remember where that was?

Yep, that was in gc/gc job called EnqueueMenuUsersForDishReminderJob

**Review by svet93 — APPROVED** [review]

**Comment by bearfact** [comment]
@svet93 you can merge this to master, rebase your branch, and then make sure to bump the version

**Comment by svet93** [comment]
👍

**Pull request merged**